Framework/Algorithms/src/WienerSmooth.cpp

```diff
@@ -369,9 +369,10 @@ WienerSmooth::smoothSingleSpectrum(API::MatrixWorkspace_sptr inputWS,
 
   if (isOddSize) {
     auto histogram = out->histogram(0);
+    auto newSize = histogram.y().size() - 1;
+
     histogram.setSharedX(inputWS->sharedX(wsIndex));
     histogram.setSharedE(inputWS->sharedE(wsIndex));
-    auto newSize = histogram.y().size() - 1;
```

Comment:
Why this change? Declaring variables close to where they are used is usually considered better.